syllabus/1-Cryptography/6-Advanced_Signatures-slides.md

```diff
@@ -202,6 +213,7 @@ Schnorr & BLS multi-signatures avoid complicating verifier logic,<br/>but introd
 - DKG protocols
 - Reduced key derivation ability
 - Verification speed
+- Proof of possession verification.
```

Comment:
I'm assuming this is referring to the part of the musig protocol that avoids chosen-public key attacks? If so, I would just cut this as it is already covered in the communication-heavy DKG protocol.

Reply:
Even for simple aggregation (when you don't need to run DKG) with BLS signatures you still can forge other participants signature if you don't show PoP.

Reply:
Ah ok, good to leave it in then.